tests/test_stats.py

```diff
@@ -97,12 +105,16 @@ def test_mean(
     array_type: ArrayType[Array], axis: Literal[0, 1, None], expected: float | list[float]
 ) -> None:
     np_arr = np.array([[1, 2, 3], [4, 5, 6]])
+    if (array_type.flags & Flags.Gpu) and np_arr.dtype.kind != "f":
+        pytest.skip("GPU arrays only support floats")
```

Comment:
is this true? I thought they supported <64 bit integers

Reply:
true, the sparse matrices only support floats! Will fix.